### PR TITLE
Onboarding design improvements

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
- make subtle button style consistent across pages
- make button position consistent on all pages
- add explainer for usernames
- fix messaging on android key setup page
- fix name validation error message
- fix small screen Use Existing page by reducing whitespace
- refactor shared style to be more minimal
- remove flow selection page / shorten
- don't suggest spellcheck bar on iOS keyboard

Tested on iOS regular and small screens, and Android regular screen size.

Video from iOS large and small: https://streamable.com/6637yt

### CODE REVIEW

File: apps/daimo-mobile/src/action/key.ts [hunk 1]
```diff
@@ -9,13 +9,13 @@ import { createEnclaveKey, loadEnclaveKey } from "../logic/enclave";
...
+      return "Device key generated in software";
     case "TRUSTED_ENVIRONMENT":
-      return "Key generated in trusted hardware";
+      return "Device key generated in trusted hardware";
     case "HARDWARE_ENCLAVE":
       return Platform.OS === "ios"
-        ? "🔒  Key generated in Secure Enclave"
-        : "🔒  Key generated in hardware enclave";
+        ? "🔒  Device key generated in Secure Enclave"
+        : "🔒  Device key generated in hardware enclave";
```
Comment (dcposch) [inline apps/daimo-mobile/src/action/key.ts:18]: idk if "Device key" is less confusing than "Key"

Comment (dcposch) [reply apps/daimo-mobile/src/action/key.ts:18]: i think we have an opportunity for a nice "Vault" ui in future. something like

> ✅ Device key · secure enclave
> ✅ Passkey backup

...then can show a partial version of the "vault" with just that first checkmark to during onboarding

user gets second checkmark later, once they add a passkey

goal of the UI is to visually communicate how our self-custody works, and assure the user their account is safe

File: apps/daimo-mobile/src/view/screen/onboarding/CreateAccountPage.tsx [hunk 2]
```diff
@@ -58,6 +59,13 @@ export function CreateAccountPage({
           <View style={{ flexDirection: "row", justifyContent: "center" }}>
             <Octicons name="person" size={40} color={color.midnight} />
           </View>
+          <Spacer h={24} />
+          <TextCenter>
+            <IntroTextParagraph>
+              Your username is what you'll go by on Daimo. Choose wisely — once
+              you get a name, you can't change it.
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/onboarding/CreateAccountPage.tsx:66]: 👍

File: apps/daimo-mobile/src/view/screen/onboarding/CreateAccountPage.tsx [hunk 3]
```diff
@@ -128,7 +137,7 @@ function NamePicker({
     } else if (result.error) {
       return <>{oct("alert")} offline?</>; // name valid, other error
     } else if (result.isSuccess && result.data) {
-      return <>{oct("alert")} sorry, that name is taken</>; // name taken
+      return <>{oct("alert")} sorry, that username is taken</>; // name taken
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/onboarding/CreateAccountPage.tsx:140]: 👍

File: apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    </View>
+  );
+}
+
+const tokenSymbol = "USDC";
+const introPages = [
+  <IntroPage title="Welcome to Daimo">
+    <IntroTextParagraph>
+      Daimo is a global payments app that runs on Ethereum. Send and receive
+      USDC on Base mainnet.
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx:77]: maybe delete second line? (i know it was there before)

File: apps/daimo-mobile/src/view/screen/onboarding/OnboardingScreen.tsx [hunk 5]
```diff
@@ -117,7 +104,6 @@ export default function OnboardingScreen({
   return (
     <View style={styles.onboardingScreen}>
       {page === "intro" && <IntroPages onNext={next} />}
-      {page === "flow-selection" && <FlowSelectionPage onNext={next} />}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/onboarding/OnboardingScreen.tsx:120]: 👍

File: packages/daimo-common/src/name.ts [hunk 6]
```diff
@@ -5,6 +5,7 @@ export function isValidName(name: string): boolean {
 export function validateName(name: string): string {
   if (name.length < 3) throw new Error("Too short");
   if (name.length > 32) throw new Error("Too long");
+  if (!/^[a-z]/.test(name)) throw new Error("Must start with lowercase letter");
```
Comment (dcposch) [inline packages/daimo-common/src/name.ts:8]: 👍

File: apps/daimo-mobile/src/view/shared/InputBig.tsx [hunk 7]
```diff
@@ -72,6 +72,7 @@ export function InputBig({
           numberOfLines={1}
           autoCapitalize="none"
           autoCorrect={false}
+          spellCheck={false}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/shared/InputBig.tsx:75]: good call

**Review by dcposch — APPROVED** [review]
LGreatTM

File: apps/daimo-mobile/src/action/key.ts (hunk 1 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/action/key.ts:18]: moved to master issue for onboarding redesign #606

File: apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx (hunk 4 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx:77]: done

**Pull request merged**